README.md

```diff
@@ -147,9 +147,10 @@ Following that, another equally important object for operating on the graph is `
 ```java
 public class GraknTweetOntologyHelper {
   public static void withGraknGraph(GraknSession session, Consumer<GraknGraph> fn) {
-    GraknGraph graphWriter = session.open(GraknTxType.WRITE);
-    fn.accept(graphWriter);
-    graphWriter.commit();
+    try (GraknGraph graphWriter = session.open(GraknTxType.WRITE)) {
```

Comment:
@sheldonkhall aah, so we have to *both* `commit` and `close` the session, is that right?